DirectProgramming/C++SYCL_FPGA/Tutorials/Features/experimental/annotated_allocation/src/annotated_class_util.hpp

```diff
@@ -0,0 +1,158 @@
+#ifndef __ANNOTATED_CLASS_UTIL_HPP__
+#define __ANNOTATED_CLASS_UTIL_HPP__
+
+#include <sycl/sycl.hpp>
+
+using namespace sycl::ext::oneapi::experimental;
+
+namespace fpga_tools {
+///////////////////////////////////////////////////////////////////////////////
+// This header provides a utility function: "alloc_annotated", which improves
+// "malloc_shared" in allocating host/share memory in the following aspects:
+//   (1) better code brevity
+//   (2) support compile-time type check
+//
+// "annotated_alloc" function takes an annotated_arg type as the only template
```

Comment:
```suggestion
// "alloc_annotated" function takes an annotated_arg type as the only template
```
please be consistent :)